packages/supabase/supabase/schema.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255),
+  email VARCHAR(255) UNIQUE,
+  "emailVerified" TIMESTAMPTZ,
+  image TEXT
+);
+CREATE TABLE accounts (
+  id SERIAL PRIMARY KEY,
+  "userId" INTEGER NOT NULL,
+  type VARCHAR(255) NOT NULL,
+  provider VARCHAR(255) NOT NULL,
+  "providerAccountId" VARCHAR(255) NOT NULL,
+  refresh_token TEXT,
+  access_token TEXT,
+  expires_at BIGINT,
+  token_type VARCHAR(255),
+  scope VARCHAR(255),
+  id_token VARCHAR(255),
+  session_state VARCHAR(255),
+  CONSTRAINT fk_user FOREIGN KEY("userId") REFERENCES users(id) ON DELETE CASCADE
+);
+CREATE TABLE sessions (
+  id SERIAL PRIMARY KEY,
+  "userId" INTEGER NOT NULL,
+  "sessionToken" VARCHAR(255) NOT NULL UNIQUE,
+  expires TIMESTAMPTZ NOT NULL,
+  CONSTRAINT fk_user FOREIGN KEY("userId") REFERENCES users(id) ON DELETE CASCADE
```

Comment:
Even when this is defined, the test fails with the following error:

> Could not find foreign keys between these entities. No relationship found between sessions and users

Here is the code that's failing:
```js
await client
        .from<AdapterSession & { user: AdapterUser }>("sessions")
        .select(`*, user: users(*)`)
        .eq("sessionToken", sessionToken)
        .single()
```

Anyone, ideas?

Reply:
How about this? 

```
-  .select(`*, user: users(*)`)
+  .select(`*, users:userId(*)`)
```

I hope this will help to solve that error.

Reply:
No, it did not seem to help, but thanks

Reply:
Just tested this:
```javascript
await supabase
      .from('sessions')
      .select(`*, user: userId(*)`)
      .eq('sessionToken', 'test_token')
      .single();
```

and I get result

```json
{
  "id": 1,
  "userId": 1,
  "sessionToken": "test_token",
  "expires": "2022-01-14T09:25:38+00:00",
  "user": {
    "id": 1,
    "name": "p3lo",
    "email": "p3lo@p3lo.com",
    "emailVerified": "2022-01-11T09:25:04+00:00",
    "image": "test"
  }
}
```

Reply:
@balazsorban44, is there any update on this? Does what @p3lo says fix the issue?